crates/typst-library/translations/ga.txt

```diff
@@ -7,5 +7,5 @@ outline = Clár Ábhair
 raw = Amhábhar
 page = Leathanach
 footnote = Nóta Coise
-email = Seoladh ríomhphoist
-telephone = Uimhir theileafóin
+email = Ríomhphost
+telephone = Teileafón
```

Comment:
is it expected that the capitalized spelling is quite different than from before? very interesting if that's how Irish works.